src/V3LinkParse.cpp

```diff
@@ -596,6 +598,73 @@ class LinkParseVisitor final : public VNVisitor {
             VL_DO_DANGLING(nodep->unlinkFrBack()->deleteTree(), nodep);
         }
     }
+    void visit(AstClocking* nodep) override {
+        m_defaultInSkewp = nullptr;
+        m_defaultOutSkewp = nullptr;
```

Comment:
VL_RESTORER on these?  (not sure)

Reply:
There are no nested clocking blocks, so I don't think any of these need `VL_RESTORER`. Unless the convention is that we always restore the default value?

Reply:
Yes, the convention is to generally restore.